File: C#/ex12/string_reverse_solution.cs
// String Reverse (Exercise 12) - Solution
// Reverse a string using C# methods.

using System;
using System.Linq;

class Program {
    static void Main() {
        string? input = Console.ReadLine();
        if (input == null) return;
        
        string reversed = new string(input.Reverse().ToArray());
        
        Console.WriteLine(reversed);
    }
}

File: C++/ex10/string_reverse_solution.cpp
// String Reverse (Exercise 10) - Solution
// Reverse a string using C++ STL.

#include <bits/stdc++.h>
using namespace std;

int main() {
    string str;
    cin >> str;
    
    reverse(str.begin(), str.end());
    
    cout << str << endl;
    
    return 0;
}
